exporter/collector/factory.go

```diff
@@ -31,14 +30,10 @@ const (
 	defaultTimeout = 12 * time.Second // Consistent with Cloud Monitoring's timeout
 )
 
-var once sync.Once
-
 // NewFactory creates a factory for the googlecloud exporter
 func NewFactory() component.ExporterFactory {
-	// register view for self-observability
-	once.Do(func() {
```

Comment:
This `once` shouldn't be necessary https://github.com/census-instrumentation/opencensus-go/blob/bf52d9df8bb2d44cad934587ab946794456cf3c8/stats/view/worker.go#L333-L335